plugins/web/opentelemetry-instrumentation-long-task/tsconfig.esm.json

```diff
@@ -5,7 +5,7 @@
     "outDir": "build/esm",
     "tsBuildInfoFile": "build/esm/tsconfig.esm.tsbuildinfo"
   },
-  "files": [ "node_modules/zone.js/dist/zone.js.d.ts"],
+  "files": [ "../../../node_modules/zone.js/dist/zone.js.d.ts"],
```

Comment:
For reference: https://github.com/open-telemetry/opentelemetry-js/pull/4257. We can avoid hard-coding the node_modules path by type references.

Reply:
Done in commit faed2143. Thanks.